index.js

```diff
@@ -2,5 +2,26 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-qunit-assert-helpers'
+  name: 'ember-qunit-assert-helpers',
+  included: function() {
+    this._ensureThisImport();
+    this.import('vendor/install-qunit-asserts.js');
```

Comment:
should use `this.import('path/here', { type: 'test' });` to ensure this goes into `dist/assets/test-support.js` instead of `dist/assets/vendor.js`.